Integrations/TinkoffIntegrationLib/TinkoffBankBroker.cs

```diff
@@ -14,32 +15,26 @@ namespace TinkoffIntegrationLib
 {
     public class TinkoffBankBroker : IBroker
     {
-        private readonly Context context;
+        private readonly SandboxContext context;
         private Action<Candle> sendCandle;
 
-        public TinkoffBankBroker(string token, int depth = 10)
+        public TinkoffBankBroker(string token)
         {
             try
             {
                 var conn = ConnectionFactory.GetSandboxConnection(token);
                 context = conn.Context;
-
-                Depth = depth;
+                context.RegisterAsync();
+                context.SetCurrencyBalanceAsync(Currency.Rub, 10000000000);
```

Comment:
maybe set a constant and a little less

Reply:
fixed

Reply:
But balance have to be big, we have a lot of users and to trade we have to use a lot of money